bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -13,66 +13,33 @@ import scala.annotation.tailrec
+
+  def calculate(progTime: TimeAmount): ObservationTimes
 
-  // This was needed for 2017A to 2017B migration as per REL-2985, as time in former proposals will be migrated to
-  // progTime, and previously, all times were based on intTime which was calculated from progTime.
-  // This has been reversed and we are no longer using integration time in the PIT as per REL-2926, making this
-  // obsolete and unused.
-//  def intTimeFromProgTime(progTime: TimeAmount): TimeAmount = {
-//    val progTimeHrs = progTime.toHours.value
-//    val numAcqs     = numAcqsFromProgTimeInHrs(progTimeHrs)
-//    val intTimeHrs  = (progTimeHrs - numAcqs * acquisitionOverhead.toHours)/(1 + otherOverheadFraction)
-//    TimeAmount(intTimeHrs, TimeUnit.HR)
-//  }
-
-  // Iterative method for calculating the program time in hours iteratively through the number of acquisitions.
-  // No longer used in transition from REL-2985 to REL-2926.
-//  private def calcProgTime(intTime: Double): Double = {
-//    val overheadTimeHrs = intTime * (1 + otherOverheadFraction)
-//
-//    @tailrec
-//    def calcProgTimeIter(numAcqsOld: Int, numAcqsNew: Int, progTime: Double, iteration: Int): Double = {
-//      // If the number of acquisitions is stable or we have
-//      if (numAcqsOld == numAcqsNew || iteration > 10)
-//        progTime
-//      else {
-//        val progTimeNew = numAcqsNew * acquisitionOverhead.toHours + overheadTimeHrs
-//        calcProgTimeIter(numAcqsNew, numAcqsFromProgTimeInHrs(progTimeNew), progTimeNew, iteration + 1)
-//      }
-//    }
-//
-//    calcProgTimeIter(0, numAcqsFromProgTimeInHrs(overheadTimeHrs), 0.0, 0)
-//  }
-//
-//  // Calculate the number of acquisitions from program time in hours.
-//  private def numAcqsFromProgTimeInHrs(progTime: Double): Int =
-//    (progTime / Overheads.visTimeHrs).toInt + ((progTime % Overheads.visTimeHrs > 0) ? 1 | 0)
 }
 
 // Due to the large number of possible blueprint bases and how each one requires different configuration params
 // to determine the overheads, it seems infeasible to read this information from a file, so for now it is hard-coded
 // and will require changing here if these values change.
 object Overheads extends (BlueprintBase => Option[Overheads]) {
-  // t_vis as per REL-2985. No longer needed by switch to REL-2926.
-//  lazy val visTime    = Hours(2)
-//  lazy val visTimeHrs = Overheads.visTime.toHours
 
-  private case class SimpleOverheads(partnerOverheadFraction: Double) extends Overheads
+  private case class SimpleOverheads(partnerOverheadFraction: Double) extends Overheads {
+    def calculate(progTime: TimeAmount): ObservationTimes = {
+      val progTimeHrs = progTime.toHours.value
+      val partTimeHrs = progTimeHrs * partnerOverheadFraction
+      ObservationTimes(TimeAmount(progTimeHrs, TimeUnit.HR), TimeAmount(partTimeHrs, TimeUnit.HR))
+    }
+  }
+  private case class TelluricOverheads(telluricTime: TimeAmount) extends Overheads {
+    val VisitLength = 1.5
```

Comment:
We're saying, I think, that we need a telluric for every 1.5 hours (or fraction thereof) of science?  If so, maybe `ScienceHoursPerTelluric` would be a better name, idk.  I was initially confused by the name.

Reply:
Sounds good. I'll also indicate the units